code/modules/html_interface/map/interactive_map.dm

```diff
@@ -81,12 +81,26 @@ var/const/ALLOW_CENTCOMM = FALSE
 /proc/generateMiniMaps()
 	set category = "Debug"
 	set name = "Generate minimaps"
-	//spawn // NO
+
+	var/last_git_hash_path = "data/last_git_commit_hash.txt"
```

Comment:
This is a new file being generated, right ? I think "data" is added to the gitignore, but can you confirm this for me, please ?

Reply:
see latest commit